libs/net/core/Converters/BooleanConverter.cs

```diff
@@ -0,0 +1,21 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace TNO.Core.Converters
+{
+    public class BooleanConverter : JsonConverter<bool>
```

Comment:
Had to add this to resolve the fact the government holiday API uses a 1 or 0 to represent Boolean values.